pipeline/outputs/elasticsearch.md

```diff
@@ -8,54 +8,59 @@ The **es** output plugin, allows to ingest your records into an [Elasticsearch](
+| Time\_Key\_Nanos | When Logstash\_Format is enabled, enabling this property sends nanosecond precision timestamps. | Off | Yes |
+| Include\_Tag\_Key | When enabled, it append the Tag name to the record. | Off | Yes |
+| Tag\_Key | When Include\_Tag\_Key is enabled, this property defines the key name for the tag. | \_flb-key | Yes |
+| Generate\_ID | When enabled, generate `_id` for outgoing records. This prevents duplicate records when retrying ES. | Off | Yes |
+| Id\_Key | If set, `_id` will be the value of the key from incoming record and `Generate_ID` option is ignored. |  | Yes |
+| Write\_Operation | The write\_operation can be any of: create (default), index, update, upsert. | create | Yes |
+| Replace\_Dots | When enabled, replace field name dots with underscore, required by Elasticsearch 2.0-2.3. | Off | Yes |
+| Trace\_Output | Print all elasticsearch API request payloads to stdout \(for diag only\) | Off | Yes |
+| Trace\_Error | If elasticsearch return an error, print the elasticsearch API request and response \(for diag only\) | Off | Yes |
+| Current\_Time\_Index | Use current time for index generation instead of message record | Off | Yes |
+| Suppress\_Type\_Name | When enabled, mapping types is removed and `Type` option is ignored. If using Elasticsearch 8.0.0 or higher - it [no longer supports mapping types](https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html), so it shall be set to On. | Off | Yes |
+| Workers | The number of [workers](../../administration/multithreading.md#outputs) to perform flush operations for this output. | `2` | No |
+| Upstream | If plugin will connect to an _Upstream_ instead of a simple host, this property defines the absolute path for the Upstream configuration file, for more details about this refer to the [Upstream Servers](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md) documentation section. | | No |
 
 > The parameters _index_ and _type_ can be confusing if you are new to Elastic, if you have used a common relational database before, they can be compared to the _database_ and _table_ concepts. Also see [the FAQ below](elasticsearch.md#faq)
 
 ### TLS / SSL
 
 Elasticsearch output plugin supports TLS/SSL, for more details about the properties available and general configuration, please refer to the [TLS/SSL](../../administration/transport-security.md) section.
 
+### AWS Sigv4 Authentication and Upstream Servers
+
+http_proxy, no_proxy and TLS parameters used for AWS Sigv4 Authentication - for connection of plugin to AWS to generate authentication signature - are never picked from NODE section of [Upstream](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md) configuration.
+TLS parameters for connection of plugin to Elasticsearch **can** be overridden in NODE section of Upstream (even if AWS authentication is used).
```

Comment:
```suggestion
The `http_proxy`, `no_proxy`, and `TLS` parameters used for AWS Sigv4 Authentication
(for connection of plugin to AWS to generate authentication signature) are never
picked from the `NODE` section of the
[Upstream](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md)
configuration. However, `TLS` parameters for connection of the plugin to
Elasticsearch **can** be overridden in the `NODE` section of Upstream, even if AWS
authentication is used.
```

Reply:
Suggestion was applied as part of f28a9c6a402fcf5dc87b1aed0902e1dff09c6749.